packages/flutter_android_lifecycle/README.md

```diff
@@ -0,0 +1,43 @@
+# Flutter Android Lifecycle Plugin
+
+[![pub package](https://img.shields.io/pub/v/flutter_android_lifecycle.svg)](https://pub.dartlang.org/packages/flutter_android_lifecycle)
+
+A Flutter plugin for Android to allow other Flutter plugins to access an Android `Lifecycle` object
+in the plugin's binding.
+
+The purpose of having this plugin instead of exposing an Android `Lifecycle` object in the engine's
+Android embedder plugins API is to force plugins to have a pub constraint that signifies the
```

Comment:
FYI, the term "embedder" refers to the C++ code. "Embedding" is the name given to the Java/Obj-C layers.
https://github.com/flutter/flutter/wiki/Glossary

Reply:
done